CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ We use [semantic versioning](http://semver.org/):
 
 # Next Release
 
+# 21.7.1
+- [feature] Option `obfuscate-security-related-outputs` to enable or disable obfuscating of security related outputs such as the access key.
+- [fix] Obfuscating the access key when logging it or printing it to the console.  
```

Comment:
I think mentioning it under feature is enough. The way you did it creates the impression that two tickets have been closed.